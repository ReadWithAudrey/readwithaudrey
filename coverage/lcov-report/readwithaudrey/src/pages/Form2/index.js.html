<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for readwithaudrey/src/pages/Form2/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">readwithaudrey/src/pages/Form2</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>5/5</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react'
import { Link, graphql } from 'gatsby'
import PropTypes from 'prop-types'
&nbsp;
import {
  InputBox,
  Label,
  Button,
  StatusBar,
  DropDownBox,
  RankingBox,
  RadioButton,
  TextBox,
  Layout,
} from '../../components/'
&nbsp;
const Form2 = ({ data }) =&gt; {
  const { q1, q2, q3, q4, q5, q6 } = data.markdownRemark.frontmatter
  return (
    &lt;Layout&gt;
      &lt;form method="POST" action="http://localhost:5000/formPart1"&gt;
        &lt;h1 className="f2 pink tc montserrat mb3 mt4"&gt;Further Details&lt;/h1&gt;
        &lt;StatusBar&gt;
          &lt;Link to="/Form1/"&gt;1. Basic Details&lt;/Link&gt;
        &lt;/StatusBar&gt;
        &lt;StatusBar type="active"&gt;2. Further Details&lt;/StatusBar&gt;
        &lt;StatusBar&gt;
          &lt;Link to="/Form3/"&gt;3. Your Bio&lt;/Link&gt;
        &lt;/StatusBar&gt;
        &lt;TextBox&gt;
          In order to find the best reading partner for you, we need a little
          bit more information
        &lt;/TextBox&gt;
        &lt;Label&gt;{q1}&lt;/Label&gt;
        &lt;RadioButton&gt;Male&lt;/RadioButton&gt;
        &lt;RadioButton&gt;Female&lt;/RadioButton&gt;
        &lt;RadioButton&gt;Prefer not to say&lt;/RadioButton&gt;
        &lt;Label&gt;{q2}&lt;/Label&gt;
        &lt;InputBox placeholder="Age" /&gt;
        &lt;Label&gt;{q3}&lt;/Label&gt;
        &lt;InputBox placeholder="Country" /&gt;
        &lt;Label&gt;{q4}&lt;/Label&gt;
        &lt;DropDownBox /&gt;
        &lt;Label&gt;{q5}&lt;/Label&gt;
        &lt;RadioButton&gt;Read&lt;/RadioButton&gt;
        &lt;RadioButton&gt;Listen&lt;/RadioButton&gt;
        &lt;RadioButton&gt;Both&lt;/RadioButton&gt;
        &lt;Label&gt;{q6}&lt;/Label&gt;
        &lt;TextBox&gt;
          Please rank the the options below in order - favourite 1st
        &lt;/TextBox&gt;
        &lt;RankingBox placeholder="1"&gt;Several fantastic short stories&lt;/RankingBox&gt;
        &lt;RankingBox placeholder="2"&gt;
          A wonderful novella (short novel)
        &lt;/RankingBox&gt;
        &lt;RankingBox placeholder="3"&gt;
          A fascinating work of non-fiction
        &lt;/RankingBox&gt;
        &lt;Link to="/Form3" className="no-underline"&gt;
          &lt;Button type="register"&gt;Continue&lt;/Button&gt;
        &lt;/Link&gt;
      &lt;/form&gt;
      &lt;TextBox&gt;
        &lt;Link to="/" className="no-underline"&gt;Go back to the homepage&lt;/Link&gt;
      &lt;/TextBox&gt;
    &lt;/Layout&gt;
  )
}
&nbsp;
Form2.propTypes = {
  data: PropTypes.object,
}
&nbsp;
export const query = graphql`
  query {
    site {
      siteMetadata {
        title
      }
    }
    markdownRemark(frontmatter: { title: { eq: "Form Part 2" } }) {
      frontmatter {
        q1
        q2
        q3
        q4
        q5
        q6
      }
    }
  }
`
&nbsp;
export default Form2</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 12 2018 12:51:26 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
