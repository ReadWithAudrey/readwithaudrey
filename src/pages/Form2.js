import React from 'react'
import { Link, graphql } from 'gatsby'
import PropTypes from 'prop-types'

import {
  InputBox,
  Label,
  Button,
  StatusBar,
  DropDownBox,
  RankingBox,
  RadioButton,
  TextBox,
  Layout,
} from '../components/'

const Form2 = ({ data }) => {
  const { q1, q2, q3, q4, q5, q6 } = data.markdownRemark.frontmatter
  return (
    <Layout className="form-padding">
      <form method="POST" action="http://localhost:5000/formPart1">
        <h1 className="f3 pink tc montserrat mb3">Further Details</h1>
        <StatusBar>
          <Link to="/Form1">1. Basic Details</Link>
        </StatusBar>
        <StatusBar type="active">2. Further Details</StatusBar>
        <StatusBar>
          <Link to="/Form3">3. Your Bio</Link>
        </StatusBar>
        <TextBox>
          In order to find the best reading partner for you, we need a little
          bit more information
        </TextBox>
        <Label>{q1}</Label>
        <RadioButton>Male</RadioButton>
        <RadioButton>Female</RadioButton>
        <RadioButton>Prefer not to say</RadioButton>
        <Label>{q2}</Label>
        <InputBox placeholder="Age" />
        <Label>{q3}</Label>
        <InputBox placeholder="Country" />
        <Label>{q4}</Label>
        <DropDownBox />
        <Label>{q5}</Label>
        <RadioButton>Read</RadioButton>
        <RadioButton>Listen</RadioButton>
        <RadioButton>Both</RadioButton>
        <Label>{q6}</Label>
        <TextBox>
          Please rank the the options below in order - favourite 1st
        </TextBox>
        <RankingBox placeholder="1">Several fantastic short stories</RankingBox>
        <RankingBox placeholder="2">
          A wonderful novella (short novel)
        </RankingBox>
        <RankingBox placeholder="3">
          A fascinating work of non-fiction
        </RankingBox>
        <Link to="/Form3">
          <Button type="register">Continue</Button>
        </Link>
      </form>
      <TextBox>
        <Link to="/">Go back to the homepage</Link>
      </TextBox>
    </Layout>
  )
}

Form2.propTypes = {
  data: PropTypes.object,
}

export const query = graphql`
  query {
    site {
      siteMetadata {
        title
      }
    }
    markdownRemark(frontmatter: { title: { eq: "Form Part 2" } }) {
      frontmatter {
        q1
        q2
        q3
        q4
        q5
        q6
      }
    }
  }
`

export default Form2
