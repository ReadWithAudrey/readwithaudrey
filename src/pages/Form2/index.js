import React from 'react'
import { Link, graphql } from 'gatsby'
import PropTypes from 'prop-types'
import { SignupContext } from '../../contexts/SignupContext'

import {
  InputBox,
  Label,
  Button,
  StatusBar,
  DropDownBox,
  RadioButton,
  TextBox,
  Layout,
  FormSection,
} from '../../components/'

const Form2 = ({ data }) => {
  const {
    heading,
    description,
    q1,
    q1Placeholder,
    q2,
    q2Placeholder,
    q3,
    readlisten1,
    readlisten2,
    readlisten3,
    q4,
    booktype1,
    booktype2,
    booktype3,
    q5,
  } = data.markdownRemark.frontmatter
  return (
    <Layout>
      <h1 className="f2 pink tc montserrat mb3 mt4">{heading}</h1>
      <FormSection>
        <Link to="/form1/">
          <StatusBar>1. Contact Details</StatusBar>
        </Link>
        <StatusBar type="active">2. Further Details</StatusBar>
        <Link to="/form3/">
          <StatusBar>3. Your Story</StatusBar>
        </Link>
        <TextBox>{description}</TextBox>
        <SignupContext.Consumer>
          {({ gender, age, readlisten, updateForm }) => (
            <form method="POST" action="http://localhost:5000/formPart1">
              <Label>{q1}</Label>
              <InputBox
                placeholder={q1Placeholder}
                onChange={updateForm}
                name="gender"
                value={gender}
              />
              <Label>{q2}</Label>
              <InputBox
                placeholder={q2Placeholder}
                onChange={updateForm}
                name="age"
                value={age}
              />
              <Label>{q3}</Label>
              <DropDownBox onChange={updateForm} name="timezone" />
              <Label>{q4}</Label>
              <RadioButton
                name="readlisten"
                onChange={updateForm}
                value={readlisten}
              >
                {readlisten1}
              </RadioButton>
              <RadioButton
                name="readlisten"
                onChange={updateForm}
                value={readlisten}
              >
                {readlisten2}
              </RadioButton>
              <RadioButton
                name="readlisten"
                onChange={updateForm}
                value={readlisten}
              >
                {readlisten3}
              </RadioButton>
              <Label>{q5}</Label>
              <RadioButton name="booktype" onChange={updateForm}>
                {booktype1}
              </RadioButton>
              <RadioButton name="booktype" onChange={updateForm}>
                {booktype2}
              </RadioButton>
              <RadioButton name="booktype" onChange={updateForm}>
                {booktype3}
              </RadioButton>
              <Link to="/form3" className="no-underline">
                <Button type="register">Continue</Button>
              </Link>
            </form>
          )}
        </SignupContext.Consumer>
      </FormSection>
    </Layout>
  )
}

Form2.propTypes = {
  data: PropTypes.object,
}

export const query = graphql`
  query {
    site {
      siteMetadata {
        title
      }
    }
    markdownRemark(frontmatter: { title: { eq: "Form Part 2" } }) {
      frontmatter {
        heading
        description
        q1
        q1Placeholder
        q2
        q2Placeholder
        q3
        readlisten1
        readlisten2
        readlisten3
        q4
        booktype1
        booktype2
        booktype3
        q5
      }
    }
  }
`

export default Form2
