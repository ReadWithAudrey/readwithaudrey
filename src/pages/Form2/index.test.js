import React from 'react'
import renderer from 'react-test-renderer'
import Form2 from './index.js'

describe('Form2 page snapshot test', () =>
  it('renders correctly', () => {
    const data = {
      site: {
        siteMetadata: {
          title: 'Form Part 2',
        },
      },
      markdownRemark: {
        frontmatter: {
          q1: 'Gender',
          q2: 'Age',
          q3: 'Location',
          q4: 'Time Zone',
          q5: 'Reader, Listener or Both?',
          q6: 'What would you like to read?',
        },
      },
    }
    const tree = renderer.create(<Form2 data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
