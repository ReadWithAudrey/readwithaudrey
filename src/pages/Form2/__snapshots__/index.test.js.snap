// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form2 page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 pv2 montserrat white bg-green w-100 f3 gzTCKs"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <Link
        to="/"
      >
        <img
          alt="book icon"
          className="h2 ph2"
          src="test-file-stub"
        />
      </Link>
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <nav
      className="sc-bwzfXH w-65 w-40-l montserrat f4 gQAWUv"
    >
      <section
        className="sc-htpNat montserrat bRYTxq"
      >
        <Link
          to="/story/"
        >
          Why?
        </Link>
        <Link
          to="/faq/"
        >
          FAQ
        </Link>
        <Link
          to="/contact/"
        >
          Contact
        </Link>
        <Link
          className="mv2 white br2 ba bw1 b--white"
          to="/Form1"
        >
          Join Us
        </Link>
      </section>
      <section
        className="sc-bxivhb igYaY"
      >
        <a
          className="dtc fr tr v-mid dn-ns p0-ns link dim white"
          onClick={[Function]}
        >
          <span
            className="db w2 mt1 mb2 bw2 pb2 bt bb b--white"
          />
          <span
            className="db w2 mt1 bw2 pb1 bt b--white"
          />
        </a>
      </section>
    </nav>
  </header>,
  <div
    className="flex flex-column items-center justify-center page-padding ph4"
  >
    <h1
      className="f2 pink tc montserrat mb3 mt4"
    >
      Further Details
    </h1>
    <section
      className="sc-dxgOiQ b--green cGznzi"
    >
      <Link
        to="/Form1/"
      >
        <section
          className="sc-gqjmRU flex items-center fl w-third montserrat bg-moon-gray fw4 f5 mb3 bypQGG"
        >
          1. Basic Details
        </section>
      </Link>
      <section
        className="sc-VigVT fl w-third bg-green montserrat ehcWMy sc-gqjmRU flex items-center fl w-third montserrat bg-moon-gray fw4 f5 mb3 bypQGG"
      >
        2. Further Details
      </section>
      <Link
        to="/Form3/"
      >
        <section
          className="sc-gqjmRU flex items-center fl w-third montserrat bg-moon-gray fw4 f5 mb3 bypQGG"
        >
          3. Your Bio
        </section>
      </Link>
      <p
        className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
      >
        In order to find the best reading partner for you, we need a little bit more information
      </p>
      <form
        action="http://localhost:5000/formPart1"
        method="POST"
      >
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          Gender
        </label>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Male"
            type="radio"
            value="Male"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Male
          </span>
        </section>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Female"
            type="radio"
            value="Female"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Female
          </span>
        </section>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Prefer not to say"
            type="radio"
            value="Prefer not to say"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Prefer not to say
          </span>
        </section>
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          Age
        </label>
        <input
          className="sc-gzVnrw montserrat mv2 pa2 w-100 ildXbv"
          placeholder="Age"
        />
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          Location
        </label>
        <input
          className="sc-gzVnrw montserrat mv2 pa2 w-100 ildXbv"
          placeholder="Country"
        />
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          Time Zone
        </label>
        <select
          className="sc-jTzLTM montserrat w-100  mv2 pa2 w-90 gFxkFW"
        >
          <option
            value="-12"
          >
            (GMT-12:00) International Date Line West
          </option>
          <option
            value="-11"
          >
            (GMT-11:00) Midway Island, Samoa
          </option>
          <option
            value="-10"
          >
            (GMT-10:00) Hawaii
          </option>
          <option
            value="-9"
          >
            (GMT-09:00) Alaska
          </option>
          <option
            value="-8"
          >
            (GMT-08:00) Pacific Time (US & Canada), Tijuana
          </option>
          <option
            value="-7"
          >
            (GMT-07:00) Mountain Time (US & Canada),Arizona
          </option>
          <option
            value="-6"
          >
            (GMT-06:00) Central Time, Mexico City, Central America
          </option>
          <option
            value="-5"
          >
            (GMT-05:00) Eastern Time (US & Canada), Lima, Bogota, Quito
          </option>
          <option
            value="-4"
          >
            (GMT-04:00) Atlantic Time (Canada), Caracus, Santiago
          </option>
          <option
            value="-3.5"
          >
            (GMT-03:30) Newfoundland
          </option>
          <option
            value="-3"
          >
            (GMT-03:00) Buenos Aires, Montevideo, Greenland, Brasilia
          </option>
          <option
            value="-2"
          >
            (GMT-02:00) Mid-Atlantic
          </option>
          <option
            value="-1"
          >
            (GMT-01:00) Azores, Cape Verde Is.
          </option>
          <option
            selected="selected"
            value="0"
          >
            (GMT+00:00) Greenwich Mean Time : Dublin, Edinburgh, Lisbon, London
          </option>
          <option
            value="1"
          >
            (GMT+01:00) Amsterdam, Paris, Brussels, Rome, Berlin, Madrid
          </option>
          <option
            value="2"
          >
            (GMT+02:00) Amman
          </option>
          <option
            value="2"
          >
            (GMT+02:00) Athens, Amman, Beirut, Bucharest, Istanbul
          </option>
          <option
            value="3"
          >
            (GMT+03:00) Kuwait, Riyadh, Baghdad
          </option>
          <option
            value="3"
          >
            (GMT+03:00) Moscow, St. Petersburg, Kuwait, Nairobi
          </option>
          <option
            value="3.5"
          >
            (GMT+03:30) Tehran
          </option>
          <option
            value="4"
          >
            (GMT+04:00) Abu Dhabi, Muscat, Baku
          </option>
          <option
            value="4.5"
          >
            (GMT+04:30) Kabul
          </option>
          <option
            value="5"
          >
            (GMT+05:00) Islamabad, Karachi, Tashkent
          </option>
          <option
            value="5.5"
          >
            (GMT+05:30) Chennai, Kolkata, Mumbai, New Delhi
          </option>
          <option
            value="5.75"
          >
            (GMT+05:45) Kathmandu
          </option>
          <option
            value="6"
          >
            (GMT+06:00) Almaty, Astana, Novosibirsk
          </option>
          <option
            value="6.5"
          >
            (GMT+06:30) Yangon (Rangoon)
          </option>
          <option
            value="7"
          >
            (GMT+07:00) Bangkok, Hanoi, Jakarta
          </option>
          <option
            value="8"
          >
            (GMT+08:00) Beijing, Kuala Lumpur, Hong Kong, Singapore
          </option>
          <option
            value="9"
          >
            (GMT+09:00) Tokyo, Seoul, Osaka
          </option>
          <option
            value="9.5"
          >
            (GMT+09:30) Adelaide, Darwin
          </option>
          <option
            value="10"
          >
            (GMT+10:00) Brisbane, Canberra, Melbourne, Sydney
          </option>
          <option
            value="11"
          >
            (GMT+11:00) Magadan, Solomon Is., New Caledonia
          </option>
          <option
            value="12"
          >
            (GMT+12:00) Auckland, Wellington, Fiji
          </option>
          <option
            value="13"
          >
            (GMT+13:00) Nukualofa
          </option>
        </select>
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          Reader, Listener or Both?
        </label>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Read"
            type="radio"
            value="Read"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Read
          </span>
        </section>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Listen"
            type="radio"
            value="Listen"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Listen
          </span>
        </section>
        <section
          className="sc-fjdhpX montserrat fw4 f5 pv2 eQHpIT"
        >
          <input
            className="sc-jzJRlG fsyDfV"
            name="Both"
            type="radio"
            value="Both"
          />
          <span
            className="sc-cSHVUG bVIEwX"
          >
            Both
          </span>
        </section>
        <label
          className="sc-htoDjs pink montserrat fw9 f5 pv2 kGbKV"
        >
          What would you like to read?
        </label>
        <p
          className="sc-kAzzGY montserrat f5 tc lh-copy jdrnSq"
        >
          Please rank the the options below in order - favourite 1st
        </p>
        <section
          className="sc-chPdSV montserrat f5 lh-copy mb3 gPRYKl"
        >
          <input
            className="sc-kgoBCf hqvUHk"
            placeholder="1"
          />
          <span
            className="sc-kGXeez ml1 clAZHx"
          >
            Several fantastic short stories
          </span>
        </section>
        <section
          className="sc-chPdSV montserrat f5 lh-copy mb3 gPRYKl"
        >
          <input
            className="sc-kgoBCf hqvUHk"
            placeholder="2"
          />
          <span
            className="sc-kGXeez ml1 clAZHx"
          >
            A wonderful novella (short novel)
          </span>
        </section>
        <section
          className="sc-chPdSV montserrat f5 lh-copy mb3 gPRYKl"
        >
          <input
            className="sc-kgoBCf hqvUHk"
            placeholder="3"
          />
          <span
            className="sc-kGXeez ml1 clAZHx"
          >
            A fascinating work of non-fiction
          </span>
        </section>
        <Link
          className="no-underline"
          to="/Form3"
        >
          <button
            className="sc-gZMcBi bg-pink hover-bg-orange white pa2 br2 w-50 h3 f3 mt4 center gTtpNx"
          >
            Continue
          </button>
        </Link>
      </form>
    </section>
  </div>,
  <footer
    className="sc-EHOje flex flex-column justify-center items-center montserrat bg-green w-100 bt bw2 b--green jvzENE"
  >
    <span
      className="sc-bZQynM flex justify-around items-center bg-white black w-100  pa2 jrqyAQ"
    >
      © Audrey
      <Link
        className="no-underline black ph1"
        to="/terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/privacy/"
      >
        Privacy
      </Link>
      <Link
        className="no-underline black ph1"
        to="/contact/"
      >
        Contact
      </Link>
    </span>
  </footer>,
]
`;
